check-license.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
accidental mix?

Reply:
Intentional- #! lines are supposed to be on the first line so it can be run with ./abc.sh (it’ll run the file with whatever executable is on that first line)

Reply:
oh, i mean do you accidentally mix this file in, cause i used to see it in a different PR, but whatever...